scripts/release.py

```diff
@@ -77,11 +77,11 @@ def _decide_next_release_number(base: str, candidate: bool = False) -> int:
 def _get_rc_parameters(dd_repo, base: str, rc, patch) -> ReleaseParameters:
     """Build a ReleaseParameters object representing the in-progress release candidate"""
     # patch value should always be 0 if we're doing an RC
-    name = "%s.0" % (base)
     new_rc_version = _decide_next_release_number(base, candidate=True)
     release_branch = DEFAULT_BRANCH if new_rc_version == 1 else base
     rn = clean_release_notes(generate_release_notes(release_branch))
-    return ReleaseParameters(release_branch, "%s.0rc%s" % (base, str(new_rc_version)), "v%s" % name, dd_repo, rn, True)
+    version_string = "%s.0rc%s" % (base, str(new_rc_version))
+    return ReleaseParameters(release_branch, version_string, "v%s" % version_string, dd_repo, rn, True)
```

Comment:
I think defining `tag_name` (or something) and returning it might be better for readability, eg:

```suggestion
    tag_name = "v%s" % version_string

    return ReleaseParameters(release_branch, version_string, tag_name, dd_repo, rn, True)
```